pocket-notes/index.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Pocket Notes</title>
+
+    <!--Link of external css files-->
+    <link rel="stylesheet" href="css/reset.css" />
+    <link rel="stylesheet" href="css/styles.css" />
+
+    <!--Link of font awesome cdn-->
+    <script
+      src="https://kit.fontawesome.com/04202a2ef3.js"
+      crossorigin="anonymous"
+    ></script>
+  </head>
+  <body>
+    <!--Start of wrapper-->
+    <div class="wrapper">
+      <!--Start of header-->
+      <header class="header">
+        <!--Start of container-->
+        <div class="container">
+          <h3>POCKET NOTES</h3>
+          <!--Start of view selection -->
+          <div class="view-selection">
+            <p>Default View</p>
+            <!--Start of select -->
+            <select name="view-selection-dropdown" id="viewBySelection">
+              <option value="default">5 Column Format</option>
+              <option value="2">2 Column Format</option>
+            </select>
+            <!--End of select -->
+          </div>
+          <!--End of view selection -->
+        </div>
+        <!--End of container-->
+      </header>
+      <!--End of header-->
+      <!--Start of notes wrapper-->
+      <div class="notes-wrapper container">
+        <div class="addNote">
+          <button class="addNoteBtn" id="addNoteBtn" onclick="addNote()">
+            <i class="fa fa-solid fa-plus"></i>
+          </button>
+        </div>
+      </div>
+      <!--End of notes wrapper-->
+      <!-- Start of modal-container-->
+      <div class="modal-container">
+        <!--Start of modal-->
+        <div class="modal">
+          <!--Start of modal header-->
+          <div class="modal-header">
+            <h4></h4>
+          </div>
+          <!--End of modal header-->
+          <!--Start of form-->
+          <form action="" id="addNoteForm">
```

Comment:
Is the form tag necessary ? It makes sense for add note but irrelevant for delete note.